app/ports/views.py

```diff
@@ -279,6 +279,7 @@ def stats(request):
     submissions_last_x_days = Submission.objects.filter(timestamp__range=[start_date, end_date]).order_by('user', '-timestamp').distinct('user')
     submissions_unique = Submission.objects.filter(id__in=Subquery(submissions_last_x_days.values('id')))
     macports_distribution = submissions_unique.values('macports_version').annotate(num=Count('macports_version'))
+    os_version_and_clt_version = sort_list_of_dicts_by_version(list(submissions_unique.values('clt_version', 'os_version').annotate(num=Count('user_id', distinct=True))), 'os_version')
```

Comment:
Somehow the versions end up sorted as `9, 8, 6, 11, 10` ...

Reply:
I am sorry, which sorting are we referring to, OS Versions or CLT Versions? For me, both are sorted very well.

Reply:
Clt versions are not sorted correctly for me on http://ec2-52-34-234-111.us-west-2.compute.amazonaws.com/statistics/ for example.